tensorflow/core/graph/mkl_graph_util.h

```diff
@@ -299,7 +299,9 @@ static inline bool IsMklOp(const string& op_name, DataType T) {
 
 static inline bool IsMklOp(const Node* n) {
   DataType T;
-  return GetNodeAttr(n->def(), "T", &T).ok() && IsMklOp(n->type_string(), T);
+  bool isOpMkl = false;
+  isOpMkl = GetNodeAttr(n->def(), "T", &T).ok() && IsMklOp(n->type_string(), T);
+  return isOpMkl;
```

Comment:
What do we accomplish by storing the results in a bool variable first?

Reply:
Thank you. This was an interesting why Static Analysis is complaining!  The message below is copied below for reference.

_"Non-void function does not return value
/jenkins/workspace/klocwork_scan/repo-to-scan/tensorflow/core/graph/mkl_graph_util.h:284 | IsMklOp()"_

Reply:
![image](https://user-images.githubusercontent.com/38869685/136079295-ff13d1e9-7fac-4d4e-92be-9b25580cc007.png)